{
  try {
    i(message,args);
  }
 catch (  Exception e) {
    i("failed to log message: " + message);
  }
}
